components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -415,6 +424,43 @@ void BraveVpnServiceDesktop::LoadCachedRegionData() {
   }
 }
 
+void BraveVpnServiceDesktop::OnPrepareCredentialsPresentation(
+    const std::string& credential_as_cookie) {
+  // Credential is returned in cookie format.
```

Comment:
you have a sequence check on all the other methods so why not here also?

Reply:
Added in https://github.com/brave/brave-core/pull/10847/commits/db48260dd1e9b819e4d991e99f8dcb3c31ca364c